src/prisma/schema.prisma

```diff
@@ -417,3 +419,25 @@ enum OrgSubscriptionStatus {
   ACTIVE
   CANCELLED
 }
+
+enum CategoryType {
+  AI_MODELS
+  PRODUCTIVITY
+  LEARNING_DEVELOPMENT
+  MARKETING
+  SALES
+  INFORMATION_TECHNOLOGY
+  ENGINEERING
+  HUMAN_RESOURCES
+  ACCOUNTING_FINANCE
+}
+
+model AICategoryType {
+  ai           AI         @relation(fields: [aiId], references: [id])
+  aiId         String     @map(name: "ai_id")
+  categoryName CategoryType     @map(name: "category_type")
```

Comment:
Rename the prisma field name to categoryType so that's it's consistent with the column name
```suggestion
  categoryType CategoryType     @map(name: "category_type")
```